src/test/suite/completion.util.ts

```diff
@@ -27,14 +28,19 @@ export const getDocUri = (p: string) => {
 };
 
 export async function checkExpectedCompletion(docUri: vscode.Uri, position: vscode.Position, expectedCompletion: vscode.CompletionItem) {
-    let hasExpectedCompletion = false;
-    await waitUntil(() => {
-        // Executing the command `vscode.executeCompletionItemProvider` to simulate triggering completion
-        (vscode.commands.executeCommand('vscode.executeCompletionItemProvider', docUri, position)).then(value => {
-            let actualCompletionList = value as vscode.CompletionList;
-            const actualCompletionLabelList = actualCompletionList.items.map(c => { return c.label; });
-            hasExpectedCompletion = actualCompletionLabelList.includes(expectedCompletion.label);
-        });
-        return hasExpectedCompletion;
-    }, 10000, 500);
+	let hasExpectedCompletion = false;
+	let actualCompletionLabelList;
+	try {
+		await waitUntil(() => {
+			// Executing the command `vscode.executeCompletionItemProvider` to simulate triggering completion
+			(vscode.commands.executeCommand('vscode.executeCompletionItemProvider', docUri, position)).then(value => {
+				let actualCompletionList = value as vscode.CompletionList;
```

Comment:
if value is of type vscode.CompletionList I would define that type directly for value.

Reply:
see https://issues.redhat.com/browse/FUSETOOLS2-430